lectures/markov_chains_I.md

```diff
@@ -726,7 +740,7 @@ We will come back to this when we introduce irreducibility in the {doc}`next lec
 
 ### Example
 
-Recall our model of the employment/unemployment dynamics of a particular worker {ref}`discussed above <mc_eg1>`.
+Recall our model of the employment/unemployment dynamics of a particular worker discussed in {ref}`Example 2: Unemployment <mc_eg1>`.
```

Comment:
@Jiarui-ZH can you update these links to use

https://sphinx-proof.readthedocs.io/en/latest/syntax.html#referencing-examples